test/Unit/Task/DeptracTest.php

```diff
@@ -25,6 +25,7 @@ public function provideConfigurableOptions(): iterable
         yield 'defaults' => [
             [],
             [
+                'cache_file' => null,
```

Comment:
I think a test with the value set should be added, in `provideExternalTaskRuns` I think

Reply:
Sure! Like this, @yguedidi? https://github.com/phpro/grumphp/pull/1023/commits/eedbe734b5021468d1484ec8e06e921ddffc5106

Reply:
yes!